apinotes/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ add_custom_command(
     COMMAND
       "${CMAKE_COMMAND}" "-E" "copy_if_different" ${inputs} "${output_dir}/")
 
-add_custom_target("copy_apinotes"
+add_custom_target("copy_apinotes" ALL
     DEPENDS "${outputs}" "${output_dir}"
```

Comment:
Question. Why are we adding add_custom_command here? Shouldn't we be using add_custom_command_target? That caused weird dependency issues for us before.

Reply:
I'm not sure about the history of this file, I didn't write this copy logic myself. We just need this target to be added to the default build target if this directory is included. This is mostly for the standalone overlay builds.

I wasn't intending to drastically change the structure for this PR as I need it to unblock some other work, but I'd be happy to discuss a more comprehensive change in the future.

Reply:
This is only adding the target, not an associated command, so this is fine as is.